src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```
@@ -5,6 +5,7 @@
     <ExcludeCurrentFullFrameworkFromPackage>true</ExcludeCurrentFullFrameworkFromPackage>
     <EnableDefaultItems>true</EnableDefaultItems>
     <Nullable>annotations</Nullable>
+    <DefineConstants>$(DefineConstants),NO_SUPRESS_GC_TRANSITION</DefineConstants>
```

Should we use **;** instead of **,**?

```suggestion
    <DefineConstants>$(DefineConstants);NO_SUPRESS_GC_TRANSITION</DefineConstants>
```